.github/workflows/ci.yml

```
@@ -1,253 +1,130 @@
+  # Summary report
+  ci-summary:
+    name: CI Summary
     runs-on: ubuntu-latest
-    needs: [backend-tests, frontend-tests, security-scan, build-validation]
+    needs: [backend, frontend]
     if: always()
 
     steps:
-      - name: PR Quality Report
+      - name: Generate summary
         run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)